docs/devguide/Autocomplete.adoc

```diff
@@ -0,0 +1,43 @@
+==== Proposed Implementation
+The auto-complete mechanism extends NurseTraverse with 2 functionality, word completion and word suggestions. To support word suggestion of only compatible words, several lists are created and the list in operation will be switch during runtime by `AutoCompletePanel#resetList()`
```

Comment:
with 2 **functionalities**, ... , support word **suggestions** , ... , will be **switched** ...